microsoft/PowerToys
# Wox project should only produce a .dll not an exe.

## Summary of the Pull Request
1. Set output type of wox.csproj to class library
2. Remove Wox.exe from msi

 
## References


## PR Checklist
* [] Applies to #xxx
* [] CLA signed. If not, go over [here](https://cla.opensource.microsoft.com/microsoft/PowerToys) and sign the CLA
* [] Tests added/passed
* [] Requires documentation to be updated
* [] I've discussed this with core contributors already. If not checked, I'm ready to accept this work might be rejected in favor of a different grand plan. Issue number where discussion took place: #xxx


## Detailed Description of the Pull Request / Additional comments


## Validation Steps Performed
1. Build the msi.  
2. Verified that Launcher still runs. 

Its difficult to see the github diff.  This is the only change in building the msi (Product.wsx). 
![image](https://user-images.githubusercontent.com/56318517/82081279-e0de0c80-969a-11ea-8967-65255895ff8a.png)
